prober/tcp.go

```diff
@@ -94,9 +94,9 @@ func ProbeTCP(ctx context.Context, target string, module config.Module, registry
 		Name: "probe_ssl_earliest_cert_expiry",
 		Help: "Returns earliest SSL cert expiry date",
 	})
-	probeSSLLatestVerifiedChainExpiry := prometheus.NewGauge(prometheus.GaugeOpts{
-		Name: "probe_ssl_latest_verified_chain_expiry",
-		Help: "Returns latest SSL verified chain expiry date",
+	probeSSLLastChainExpiryTimestampSeconds := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "probe_ssl_last_chain_expiry_timestamp_seconds",
+		Help: "Returns last SSL chain expiry in timestamp seconds",
```

Comment:
in unixtime

Looks like the existing metric help could do with some consistency too.